### PR TITLE
Fix CentOS 7 RPM Dependencies perl, sudo, initscripts

### LINKED ISSUE #658: Fix Perl Interpreter Paths in Built Scripts
As an OpenGEE user, I want the SCons build of Open GEE to produce Perl scripts with the canonical path to the Perl interpeter (`/usr/bin/perl` on Red Hat and Ubuntu distributions).

## Steps to Reproduce

1. Build Open GEE RPMs on Cent OS 7:

```bash
cd earthenterprise/earth_enterprise
scons -j$[ 2 * $(nproc) ] release=1 build
scons -j$[ 2 * $(nproc) ] release=1 stage_install
cd rpms
./gradlew openGeeRpms
```

2. List `opengee-server` dependencies containing `perl`:

```
cd build/distributions
rpm -qpR opengee-server-5.2.0-1.el7.x86_64.rpm |grep perl

/bin/perl
/usr/bin/perl
perl >= 0:5.004
perl(AnyDBM_File)
perl(Fcntl)
perl(File::Path)
perl(Getopt::Long)
perl(strict)
```

## Expected Result

Only capabilities with the path `/usr/bin/perl` show up as requirements.

## Observed Result

Capability `/bin/perl` shows up as a requirement.

### PR DESCRIPTION
**Pull request by tst-ppenev**
* Added a rewrite filter for `find-requires` capabilities of RPM packages that map `/bin` paths to executables to `/usr/bin` paths.
* Added an analogous rewrite filter for RPM `requiresCommands`.
* Added `initscripts` as a dependency for `opengee-server`, since it bundles Apache, and Apache needs `initscripts`.

Fixes #658

### CODE REVIEW

**Comment by tst-lsavoie** [comment]
@tst-ppenev we should merge this into release_5.2.1.

**Comment by tst-ppenev** [comment]
@tst-lsavoie: Thanks. Fixed.

File: earth_enterprise/rpms/build.gradle [hunk 1]
```diff
@@ -59,8 +62,19 @@ if (new File('build.properties').exists()) {
     }
 }
 
+def openGeeVersionFile = new File("../src/NATIVE-REL-x86_64/gee_long_version.txt")
```
[the remark this answers is not in the document]
Comment (tst-ppenev) [reply earth_enterprise/rpms/build.gradle]: I think that's worse. It requires running `stage_install`, which you presumably run after a `build`.  I certainly don't want to wait for a `build` and `stage_install` to get a version string.

[the remark this answers is not in the document]
Comment (tst-jblair) [reply earth_enterprise/rpms/build.gradle]: Fair enough!

**Pull request merged**